addons/account/views/account_move_views.xml

```diff
@@ -94,6 +94,31 @@
                 <kanban class="o_kanban_mobile" create="false" group_create="false">
                     <field name="company_currency_id"/>
                     <field name="partner_id"/>
+                    <field name="payment_id"/>
```

Comment:
Why do you need it in the kanban? It is not editable, fields here are only used for display.

Am I missing something?

Reply:
I don't know how it works. I copied list of fields from another view. I could only remove field one by one and check if the mobile view still works or not, but it's boring :grimacing:

Reply:
:pray:

Reply:
well, diffs are supposed to be the minimum to fix a specific problem.... so yes, you should only include here what's fixing your issue, and not copy the whole list of fields from another view or version without "knowing how that works" :roll_eyes: